quickstart/polyglot/tests/e2e/test_polyglot_workflow/test_polyglot_workflow.py

```diff
@@ -0,0 +1,146 @@
+import os
+import unittest
+
+import pytest
+from application_sdk.test_utils.e2e.base import BaseTest
+
+
+class TestPolyglotWorkflow(unittest.TestCase, BaseTest):
+    """E2E tests for Polyglot workflow.
+
+    Polyglot is a simple workflow that:
+    - Takes a number as input
+    - Calculates factorial using Java code from Python via JPype
+    - Saves the result to JSON
+    - Does NOT have auth, metadata, or preflight check endpoints
+    - Only has: health check, start workflow, get configuration, and workflow status
+    """
+
+    extracted_output_base_path = (
+        "./local/dapr/objectstore/artifacts/apps/default/workflows"
+    )
+
+    @classmethod
+    def prepare_dir_paths(cls):
+        """
+        Prepares directory paths for the test to pick up the configuration file.
+        Overrides base method to skip schema directory check since polyglot
```

Comment:
Seeing this method overridden everywhere
We should fix the schema directory check in the sdk itself